docs/modules/operation/pages/quick-start/performance-data.adoc

```diff
@@ -22,20 +22,11 @@ We recommend that you do not use the default community string in production.
 NOTE: If you are using SNMPv3, you need to configure details for a User-Based Security Model (USM) user instead of a community string.
```

Comment:
Seeing this in context, it makes more sense for this to be a note in step 3 of the "Configure SNMP for provisioning topic", after line 38.